src/pages/createlist/components/QueryBuilder/QueryBuilder.module.css

```diff
@@ -1,3 +0,0 @@
-button[class*="fullWidth"] {
-  width: fit-content !important;
```

Comment:
Quick sanity check: Do we still need the `width: fit-content` without the `!important`?